bundles/org.openhab.binding.serial/src/main/java/org/openhab/binding/serial/internal/handler/SerialDeviceHandler.java

```diff
@@ -100,6 +102,13 @@ public void initialize() {
                             type.getId());
                     if (deviceChannel != null) {
                         channels.put(c.getUID(), deviceChannel);
+
+                        int delay = deviceChannel.getRefreshInterval();
+                        if (delay > 0) {
```

Comment:
Please document that setting refreshinterval to 0 is essential disabling the refreshing

Reply:
I've changed the description to:

> A value greater than the default of 0 will trigger a periodical "Refresh Command" for services that do not automatically push changed values to the client

Is this sufficient?